app/javascript/lib/composables/use_fuzzy_typeahead.ts

```diff
@@ -8,19 +8,19 @@ export function useFuzzyTypeahead<T extends object>({
   maxMatches,
   fuseOptions,
 }: {
-  searchables: Array<T>;
+  searchables: Ref<Array<T>>;
```

Comment:
By making this reactive, it's possible to see the boost effects on emoji searches immediately, rather than needing to refresh the page. There's actually even a live-typing effect, if one updates the boosted name while it is displayed in the search results.